### ISSUE TITLE
Self-Author-Procedure

### ISSUE THREAD

**Issue by spark-sse** [e-Learning-by-SSE/nm-self-learning#140]
https://github.com/e-Learning-by-SSE/nm-self-learning/pull/138/files#r1525997091

Die verlinkte Code passage sollte nach implementierung ebenfalls entfernt werden

**Comment by spark-sse** [comment]
@Elscha macht vermutlich sinn wenn du das übernimmst?

**Comment by staciwamonika** [comment]
Ja, die Veränderung ist schon in master [6e9d33a](https://github.com/e-Learning-by-SSE/nm-self-learning/commit/6e9d33a7f11d2b0ce7139363c5527a785c7508ad)
